### PR TITLE
copy guidance from javascript repo

### PR DESCRIPTION
**Pull request by duglin**
Copy some useful guidance docs from the Javascript repo

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: pr_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+* [Submitting and Updating a Pull Request](#submitting-and-updating-a-pull-request)
+* [Congratulations!](#congratulations)
+
+## Getting Started
+
+When creating a pull request, first fork this repository and clone it to your
+local development environment. Then add this repository as the upstream.
+
+```console
+git clone https://github.com/mygithuborg/sdk-javascript.git
```
Comment (slinkydeveloper) [inline pr_guidelines.md:21]: s/sdk-javascript/sdk-java/g

File: pr_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+git remote add upstream https://github.com/cloudevents/sdk-javascript.git
+```
+
+## Branches
+
+The first thing you'll need to do is create a branch for your work.
+If you are submitting a pull request that fixes or relates to an existing
+GitHub issue, you can use this in your branch name to keep things organized.
+For example, if you were to create a pull request to fix
+[this error with `httpAgent`](https://github.com/cloudevents/sdk-javascript/issues/48)
```
Comment (slinkydeveloper) [inline pr_guidelines.md:32]: same

File: pr_guidelines.md (hunk 1 above)
Comment (duglin) [reply pr_guidelines.md]: fixed

File: pr_guidelines.md (hunk 2 above)
Comment (duglin) [reply pr_guidelines.md]: fixed!

**Pull request merged**